reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -57,66 +98,48 @@ PowerShell Security at DerbyCon                                      Wed, 28 Sep
 DSC Resource Kit September Release                                   Thu, 22 Sep 2016 00:25:37 +0000
 PowerShell DSC and implicit remoting broken in KB3176934             Tue, 23 Aug 2016 15:07:50 +0000
 PowerShell on Linux and Open Source!                                 Thu, 18 Aug 2016 15:32:02 +0000
+
+#>
 ```
 
-This command uses the `Invoke-RestMethod` cmdlet to get information from the Windows PowerShell Blog RSS feed.
-The command uses the `Format-Table` cmdlet to display the values of the **Title** and **pubDate** properties of each blog in a table.
+This command uses the `Invoke-RestMethod` cmdlet to get information from the PowerShell Blog RSS feed.
```

Comment:
According to the Style Guide: 
- When talking about a cmdlet in paragraph, use ` to highlight cmdlet names
- Property names and parameter names should be in bold

Can you fix this in your PR?

Reply:
hyphen or no hyphen on the parameters, or does it matter?

Reply:
I don't think it matters. I think having the hyphen is useful to distinguish the difference between parameters and properties when talking about both.

Reply:
agreed. I'll fix them.

Reply:
fixed.